strawberry/arguments.py

```diff
@@ -119,8 +124,8 @@ def convert_argument(
     if value is None:
         return None
 
-    if is_unset(value):
-        return value
+    if value is _deprecated_UNSET:
+        return _deprecated_UNSET
```

Comment:
I tried to look at adding tests for this, but from what I could tell this already has tests. So I don't really know what to do

Reply:
codecov is probably wrong :D